### DISCUSSION TITLE
How to monitor self-hosted renovate

### DISCUSSION THREAD

**Discussion by chandapukiran** [renovatebot/renovate#25906] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### What is your question?

Hi, i would like to know if there is any way to monitor self-hosted renovate; may be sending alerts to slack or some metrics for prometheus/grafana. 

I don't know if it is possible so looking for some suggestion, currently we are using self-hosted renovate which runs as github action in our own runners. 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Logs (JSON formatting is available) or OpenTelemetry

**Reply by HonkingGoose** [reply]
@chandapukiran you may want to read this thread to get more ideas on how to use Opentelemetry to create a custom dashboard:

- https://github.com/renovatebot/renovate/discussions/23105

Also read the Swissquote user story, that shows a "Renovate performance dashboard" that they made:

- [Renovate docs, Swissquote user story](https://docs.renovatebot.com/user-stories/swissquote/)

You may want to subscribe to this issue:

- https://github.com/renovatebot/renovate/issues/23961
